xarray/core/datatree.py

```diff
@@ -1447,6 +1448,8 @@ def map_over_datasets(
             Function will not be applied to any nodes without datasets.
         *args : tuple, optional
             Positional arguments passed on to `func`.
```

Comment:
Should we show / state that the `*args` can be `DataTree`s while the `kwargs` cannot? This is shown clearly in the `apply_ufunc` docstring.